### PR TITLE
Implement "move card to review" for Projects V2

### LINKED ISSUE #42: Write MoveCardToReview for GraphQL
https://github.com/SonarSource/gh-action-lt-backlog/tree/master/MoveCardToReview

### PR DESCRIPTION
**Pull request by ilia-kebets-sonarsource**
Fixes #42

### CODE REVIEW

**Review by pavel-mikula-sonarsource — CHANGES_REQUESTED** [review]
Minor polishing

File: lib/ProjectContent.ts [hunk 1]
```diff
@@ -192,7 +193,7 @@ export class ProjectContentV2 extends ProjectContent {
     return null;
   }
 
-  public async moveCard(card: Card, column_id: number): Promise<void> {
+  public async moveCard(card: Card, column_id: string): Promise<void> {
```
[the remark this answers is not in the document]
Comment (ilia-kebets-sonarsource) [reply lib/ProjectContent.ts:196]: In the context of ProjectContentV2, column_id is a string, so I override it to be more specific.

File: MoveCardToReview/ReadMe.md [hunk 2]
```diff
@@ -14,13 +18,51 @@ Token to access GitHub API.
...
+### `project-number`
+
+The project number when Projects V2 is used. [This page](../docs/github.md) explains how this can be obtained.
 
 ## Outputs
 
 None
 
-## Example usage
+## Usage examples
```
Comment (pavel-mikula-sonarsource) [inline MoveCardToReview/ReadMe.md:31]: Same as on the other PR - we'll need to add the section with prerequisites mentioning ticket to REs with example link (to the correct token)

Comment (pavel-mikula-sonarsource) [inline MoveCardToReview/ReadMe.md:31]: Same as on the other PR - we'll need to add the section with prerequisites mentioning ticket to REs with example link (to the correct token)

**Review by pavel-mikula-sonarsource — CHANGES_REQUESTED** [review]
We'll need to prerequisite section here too

File: MoveCardToReview/ReadMe.md (hunk 2 above)
Comment (ilia-kebets-sonarsource) [reply MoveCardToReview/ReadMe.md:36]: done

File: MoveCardToReview/ReadMe.md [hunk 3]
```diff
@@ -28,6 +28,11 @@ The project number when Projects V2 is used. [This page](../docs/github.md) expl
 
 None
 
+## Prerequisites
+
+Send an email to REs asking to add a `github_kanban` token to the Vault configuration of your repository for each repository where you need to use this.
+A change like that [for the squad](https://github.com/SonarSource/re-terraform-aws-vault/blob/edba176cf3d89dd3a7e9ffed5807a8082076fbfe/orders/squad-jsts.yaml#L21-L29) and like that [for each repository](https://github.com/SonarSource/re-terraform-aws-vault/blob/edba176cf3d89dd3a7e9ffed5807a8082076fbfe/orders/squad-jsts.yaml#L53-L54).
```
Comment (pavel-mikula-sonarsource) [inline MoveCardToReview/ReadMe.md:34]: For the repository, I'd use `SonarJS` as a flagship repo. Seeing `switft` is a bit surprise there and ownership might move. SonarJS will for sure stay under the sonar-jsts.yml with the change for the squad part.

**Review by pavel-mikula-sonarsource — APPROVED** [review]
LGTM

**Pull request merged**